src/03-error-handling-async/index.test.ts

```diff
@@ -1,30 +1,44 @@
 // Uncomment the code below and write your tests
-// import { throwError, throwCustomError, resolveValue, MyAwesomeError, rejectCustomError } from './index';
+import {
+  throwError,
+  throwCustomError,
+  resolveValue,
+  MyAwesomeError,
+  rejectCustomError,
+} from './index';
 
 describe('resolveValue', () => {
   test('should resolve provided value', async () => {
-    // Write your test here
+    const test_data = 'test';
+    const result = await resolveValue(test_data);
+    expect(result).toBe(test_data);
```

Comment:
```suggestion
    const value = 'value';
    await expect(resolveValue(value)).resolves.toBe(value);
```

Reply:
Thank you for carefully and thoroughly checking the work and making suggestions!